src/cmd-cloud-prune

```diff
@@ -141,13 +143,13 @@ def main():
                         # S3().delete_object(args.bucket, f"{args.prefix}{str(current_build.id)}")
                     case "images":
                         raise NotImplementedError
-            if not args.dry_run:
-                build.setdefault("policy-cleanup", []).append("cloud-uploads")
-                builds_json_data["builds"][index] = build
+            build.setdefault("policy-cleanup", []).append("cloud-uploads")
+            builds[len(builds) - 1 - index] = build
 
-    if not args.dry_run:
-        # Save the updated builds.json to local builds/builds.json
-        save_builds_json(builds_json_data)
+    # Ensure the builds are ordered from newest to oldest.
+    builds.sort(key=lambda x: x['id'], reverse=True)
+    # Save the updated builds.json to local builds/builds.json
+    save_builds_json(builds_json_data, BUILDFILES['list'])
```

Comment:
Looks like we lost the conditional on `args.dry_run` here.

Reply:
If we want to see/print the updated `builds.json`, we will need to skip on the `args.dry_run` at couple of places in order to update the `builds.json`.

Reply:
Hmm, right. But I think it's also weird to have a `--dry-run` operation actually overwrite `builds.json`. But OK, I think that's fine.